roles/KubevirtTemplateValidator/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
I do not think we need this file, it won't work without the deploy manifests anyway

Reply:
Removed